core/src/test/java/com/linecorp/armeria/internal/common/ArmeriaHttpUtilTest.java

```diff
@@ -384,7 +383,8 @@ void stripTEHeadersAccountsForValueSimilarToTrailers() {
 
     @Test
     void stripTEHeadersAccountsForOWS() {
-        final io.netty.handler.codec.http.HttpHeaders in = new DefaultHttpHeaders();
+        // Disable headers validation to allow optional whitespace.
```

Comment:
What's the rational behind this?

Reply:
Previously, Netty used to allow optional whitespace in headers but now it is disallowed.
We can create a value containing whitespace by setting `validate` to false.
The value of TE is trimmed internally in `toArmeria(in, out)`. Since Netty disallows optional whitespace with the headers validation mode, we may remove this test. 

But this test would be still useful when users want to set a lower Netty version instead of the latest one.

Reply:
Thanks for the explanation. 😄